molpipeline/estimators/chemprop/abstract.py

```diff
@@ -62,10 +99,20 @@ def __init__(
                 callbacks=[],
             )
         self.lightning_trainer = lightning_trainer
-        self.batch_size = batch_size
-        self.n_jobs = n_jobs
+        self.trainer_params = get_non_default_params_trainer(self.lightning_trainer)
```

Comment:
It would be better to collect and save all parameters, not only the non-default ones. If a model is serialized and the default params change the model can not be deserialized. This might also be addressed in a future issue.

Reply:
Done